src/DatumEither.ts

```diff
@@ -525,54 +511,76 @@ export const Traversable: Traversable2<URI> = {
 }
 
 /**
- * @since 3.2.0
+ * @since 4.0.0
```

Comment:
TODO: Fix for consistency. In most places where the behavior of an existing function changed I added something like `(semantics changed in 4.0.0)` but left the original `@since`. Leaving this here to illustrate an alternative.

Reply:
I think updating the since when behavior changes is ideal since there is a deprecation in 3.5.